projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-block-editor-assets.php

```diff
@@ -214,13 +229,24 @@ function () {
 				}
 			);
 
+			// Apply filtering based on query parameter
+			$exclude_param = $request->get_param( 'exclude' );
+			$exclude_rules = $this->parse_exclude_parameter( $exclude_param );
+
+			if ( ! empty( $exclude_rules ) ) {
+				$html['styles']  = $this->filter_assets_from_html( $html['styles'], 'link', 'href', $exclude_rules );
+				$html['scripts'] = $this->filter_assets_from_html( $html['scripts'], 'script', 'src', $exclude_rules );
+			}
+
 			return rest_ensure_response(
 				array(
 					'allowed_block_types' => array_merge(
 						$this->get_core_block_types(),
 						self::ALLOWED_PLUGIN_BLOCKS
 					),
+					// @phan-suppress-next-line PhanTypePossiblyInvalidDimOffset -- Keys are guaranteed by callback above
 					'scripts'             => $html['scripts'],
+					// @phan-suppress-next-line PhanTypePossiblyInvalidDimOffset -- Keys are guaranteed by callback above
```

Comment:
These values are inaccessible to Phan due to the opaque callback passed to `with_absolute_urls`. That callback explicitly defines these keys,.